src/pages/Main/Slider/SimpleSlider.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Slider from 'react-slick';
+import S from './Styled.SimpleSlider';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const slider = [
```

Comment:
UPPER_SNAKE_CASE로 작성해서 상수 데이터임을 나타내주는 것도 좋습니다.
```suggestion
const SLIDE_LIST = [
```